Library/Formula/http-parser.rb

```diff
@@ -0,0 +1,18 @@
+class HttpParser < Formula
+  homepage "https://github.com/joyent/http-parser"
+  url "https://github.com/joyent/http-parser/archive/v2.5.0.tar.gz"
+  sha256 "e3b4ba58f4e6ee5fbec781df020e5cb74c3a799a07f059e1e125127a0b801481"
+
+  def install
+    system "make", "library", "SONAME=libhttp_parser.dylib"
+    include.install "http_parser.h"
```

Comment:
The `make install` is only suitable for Linux. It creates a `.so.2.5.0` and soft link to `.so`, and makes `brew audit --strict` fail.